srv/modules/runners/deepsea.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+"""
+This runner implements helper functions for DeepSea in general
+"""
+
+import os
+import re
+
+
+def version():
+    """
+    Returns the DeepSea version info currently installed
+    """
+    version_path = '/usr/share/doc/packages/deepsea/version.txt'
```

Comment:
A hardcoded path sounds risky to me.

On non SUSE-platforms, this would be `/usr/share/doc/deepsea/version.txt`, for example (if RPM's `%docdir` macros is utilized). Wouldn't it be better to keep it somewhat "closer" to the DeepSea code?

Reply:
Good catch. I would be in favor of storing it in the pillar. 
@swiftgist what do you think?